engine/src/cmd/unit_const_cache.h

```diff
@@ -7,8 +7,9 @@
 
 class Mesh;
 
-struct StringIntKey
+class StringIntKey
```

Comment:
As long as this code builds and runs everywhere, I'm fine with it. I think the issue with a few of the classes vs. structs was that the Microsoft linker didn't like them. But if this is an exception, then that's cool.